counter/coco.nut

```diff
@@ -1,20 +1,22 @@
 [coco]
-version = "0.3.0"
+version = "0.2.0"
 
 [module]
 name = "Counter"
-version = "1.0.0"
-license = "MIT"
-repository = "https://github.com/sarvalabs/cocolang-examples/counter"
-authors = ""
+version = "0.0.1"
+license = []
+repository = ""
+authors = []
 
 [target]
 os = "MOI"
 arch = "PISA"
 
 [target.moi]
 format = "YAML"
-output = "counter"
+output = "counter.yaml"
```

Comment:
i dont think the the output requires a file extension

Reply:
This comes from the default coco.nut file, and the default Coco version in the coco.nut file is also incorrect, I guess. It comes as 0.2.0 instead of 0.4.0. Anyways, I will modify them to the correct ones in this PR